clio-dataaccess-model/src/test/scala/org/broadinstitute/clio/server/dataaccess/util/UpsertIdGeneratorSpec.scala

```diff
@@ -0,0 +1,23 @@
+package org.broadinstitute.clio.server.dataaccess.util
+
+import org.scalatest.{FlatSpec, Matchers}
+
+class UpsertIdGeneratorSpec extends FlatSpec with Matchers {
+  behavior of "UpsertIdGenerator"
+
+  val count = 100000
+  def generateIds(): Seq[String] = Seq.fill(count) {
+    UpsertIdGenerator.nextId()
+  }
```

Comment:
Wondering: do you think it'd be a realistic test of the generator's thread-safety if we made this generate IDs inside `Future`s, then wrapped the whole thing inside a `Future.sequence`?

Reply:
I don't know.  I can try it.

Reply:
Is this what you had in mind?
``` scala
  def generateIds(): Future[Seq[String]] = Future.sequence {
    Seq.fill(count) {
      Future(UpsertIdGenerator.nextId())
    }
  }
```
I'd have to move `TestKitSuite` somewhere accessible to the dataaccess module to write the tests though.

Reply:
Yup that's what I was thinking!

I think all we'd need from the `TestKitSuite` for this is the execution context. Can you try using the default EC instead? `scala.concurrent.ExecutionContext.Implicits.global`